manifests/bosh-manifest/reference-bosh-manifest.yml

```diff
@@ -151,6 +153,50 @@ networks:
 - name: public
   type: vip
 properties:
+  collectd:
+    config: |
+      LoadPlugin cpu
+      LoadPlugin disk
+      LoadPlugin entropy
+      LoadPlugin interface
+      LoadPlugin load
+      LoadPlugin memory
+      LoadPlugin swap
+      LoadPlugin uptime
+
+      LoadPlugin df
+      <Plugin df>
+        ReportInodes true
+        ReportReserved true
+      </Plugin>
+
+      LoadPlugin syslog
+      <Plugin syslog>
+        LogLevel warning
+        NotifyLevel WARNING
+      </Plugin>
+
+      LoadPlugin vmem
+      <Plugin vmem>
+        Verbose false
+      </Plugin>
+
+      LoadPlugin "write_graphite"
+      <Plugin "write_graphite">
+       <Node "myNode">
+         #FIXME: hard coded static IP to be removed during #121602315
+         # This address comes from 040-graphite.yml:31 and has been copied
+         # as `grab` cannot interpolate in a multi-line string.
```

Comment:
It would be good to clarify that `040-graphite.yml` is from the CF manifest and not from this manifest. It's probably worth updating the comment in that file to mention that that IP is now also replicated in this manifest.

This is also another reason we can't use `grab` here (in addition to this being a multi-line string).